frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -84,6 +84,8 @@ global.people.add({
     assert_typeahead_equals("hi emoji :", false);
     assert_typeahead_equals("hi emoji :ta", emoji_list);
     assert_typeahead_equals("hi emoji :da", emoji_list);
+    assert_typeahead_equals("hi emoji :da_", emoji_list);
+    assert_typeahead_equals("hi emoji :da ", emoji_list);
```

Comment:
I don't know about node and node testing much. But I will try my best to do this! :)